examples/resizable_images/ImageShowModel.hpp

```diff
@@ -23,7 +23,7 @@ class ImageShowModel : public NodeDelegateModel
 public:
     ImageShowModel();
 
-    ~ImageShowModel() = default;
+    virtual ~ImageShowModel();
```

Comment:
`~ImageShowModel() override;`

Reply:
@Daguerreo 
Hello, Daguerreo！Thank you very much for your valuable advice, but I am a novice who has not yet started, so I can not read the advice you have given. Can you elaborate on it?